testem.js

```diff
@@ -1,13 +1,19 @@
-/*jshint node:true*/
+/* eslint-env node */
 module.exports = {
-  "framework": "qunit",
-  "test_page": "tests/index.html?hidepassed",
-  "disable_watching": true,
-  "launch_in_ci": [
-    "PhantomJS"
+  test_page: 'tests/index.html?hidepassed',
+  disable_watching: true,
+  launch_in_ci: [
+    'Chromium'
   ],
-  "launch_in_dev": [
-    "PhantomJS",
-    "Chrome"
-  ]
+  launch_in_dev: [
+    'Chromium'
+  ],
+  browser_args: {
+    Chrome: [
```

Comment:
Chrome vs Chromium? You use chromium but the args are defined for chrome. Does this work?

Reply:
It worked, just without the options (didn't run headless) :smile: Nice find! (fixed)